demos/browser/ChimeMeetings.js

```diff
@@ -0,0 +1,49 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+const AWS = require('aws-sdk');
+
+module.exports = class ChimeMeetings {
+
+    // Create an AWS SDK Chime object. Region 'us-east-1' is currently required.
```

Comment:
Why is us-east-1 required?

Reply:
I am not sure this comment was already present in handlers.js

Reply:
This was here before when the SDK was using the global API.